install/installation-debian.md

```diff
@@ -110,64 +121,87 @@ your local system using the `psql` command-line utility.
 <procedure>
 
 ### Setting up the TimescaleDB extension
+
+   Before you create a TimescaleDB extension 
+
 1.  Restart the service after enabling TimescaleDB with `timescaledb-tune`:
+
     ```bash
     systemctl restart postgresql
     ```
 
 1.  On your local system, at the command prompt, connect to the PostgreSQL
     instance as the `postgres` superuser:
+
     ```bash
-    su postgres -c psql
+    -u postgres psql
     ```
+
     If your connection is successful, you'll see a message like this, followed
     by the `psql` prompt:
-    ```
-    psql (13.3, server 12.8 (Ubuntu 12.8-1.pgdg21.04+1))
-    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+
+    ```bash
+    psql (15.0 (Ubuntu 15.0-1.pgdg20.04+1), server 14.5 (Ubuntu 14.5-2.pgdg20.04+2))
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
     Type "help" for help.
-    tsdb=>
     ```
+
+1.  Set the password for the `postgres` user using:
+
+    ```sql
+    \password postgres 
+    ```
+
+1.  Exit from PostgreSQL using the command `\q`.
```

Comment:
Can't do it as a suggestion because GH markup, but put this command in a block instead of inline.

Reply:
done.